HCP docs update for IBM Power (#92359)
openshift/openshift-docs: modules/hcp-adding-agents.adoc

Reviewer: I don't see this file in the preview

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-adding-agents_{context}"]
= Adding agents to the heterogeneous cluster

> You can add Agents by manually configuring the machine to boot with the live ISO. The live ISO may be downloaded and used to boot a node a bare-metal or a virtual machine. On boot, the node will communicate with the `assisted-service` and register as an Agent in the same namespace as the `infraenv`. Once each agent is created, optionally set its `installation_disk_id` and `hostname` in the specifications. Then approve it to indicate that the agent is ready for use.

Reviewer: the `assisted-service`, the `infraenv` <--- these need nouns after them

```suggestion
You can add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and register as an agent in the same namespace as the `infraenv`. Once each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.
```

.Procedure

> . The following command can be used to get agents:
> +
> [source,yaml]
> ----
> oc -n <hosted_control_plane_namespace> get agents
> NAME                                   CLUSTER   APPROVED   ROLE          STAGE
> 86f7ac75-4fc4-4b36-8130-40fa12602218                        auto-assign
> e57a637f-745b-496e-971d-1abbf03341ba                        auto-assign
> ----
> +
> . The following command can be used to patch the agent:
> +
> [source,yaml]
> ----
> oc -n <hosted_control_plane_namespace> patch agent 86f7ac75-4fc4-4b36-8130-40fa12602218 -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-0.example.krnl.es"}}' --type merge
> ----
> +
> . The following command can be used to patch the agent:
> +
> [source,yaml]
> ----
> oc -n <hosted_control_plane_namespace> patch agent 23d0c614-2caa-43f5-b7d3-0b3564688baa -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-1.example.krnl.es"}}' --type merge
> ----
> +
> . The following command can be used to check the agent approval status:
> +
> [source,yaml]
> ----
> oc -n <hosted_control_plane_namespace> get agents
> NAME                                   CLUSTER   APPROVED   ROLE          STAGE
> 86f7ac75-4fc4-4b36-8130-40fa12602218             true       auto-assign
> e57a637f-745b-496e-971d-1abbf03341ba             true       auto-assign
> ----

Reviewer: Format issues here, and it seems like a duplicate step:

```suggestion
. List the agents by running the following command:
+
[source,terminal]
----
$ oc -n <hosted_control_plane_namespace> get agents
----
+
.Example output
----
NAME                                   CLUSTER   APPROVED   ROLE          STAGE
86f7ac75-4fc4-4b36-8130-40fa12602218                        auto-assign
e57a637f-745b-496e-971d-1abbf03341ba                        auto-assign
----

. Patch an agent by running the following command:
+
[source,terminal]
----
$ oc -n <hosted_control_plane_namespace> patch agent 86f7ac75-4fc4-4b36-8130-40fa12602218 -p '{"spec":{"installation_disk_id":"/dev/sda","approved":true,"hostname":"worker-0.example.krnl.es"}}' --type merge
----

. Check the agent approval status by running the following command:
+
[source,terminal]
----
$ oc -n <hosted_control_plane_namespace> get agents
----
+
.Example output
----
NAME                                   CLUSTER   APPROVED   ROLE          STAGE
86f7ac75-4fc4-4b36-8130-40fa12602218             true       auto-assign
e57a637f-745b-496e-971d-1abbf03341ba             true       auto-assign
----
```

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-adding-agents_{context}"]
= Adding agents to the heterogeneous cluster

> You can add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and register as an agent in the same namespace as the `infraenv`. Once each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.

Reviewer 2: If this task is required, you might remove the _can_ here, as it makes it sound like the task is optional. 

```suggestion
You add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and register as an agent in the same namespace as the `infraenv`. Once each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.
```

> You can add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and register as an agent in the same namespace as the `infraenv`. Once each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.

Reviewer 3: Use "once" only when meaning "one time".

```suggestion
You can add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and registers as an agent in the same namespace as the `infraenv`. When each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.
```
